app/views/pages/_contact.haml

```diff
@@ -6,7 +6,29 @@
         %hr.star-primary
     .row
       .col-lg-8.col-lg-offset-2
-        %p= t('contact_intro_html')
+        .panel.panel-danger#confirm_contact
+          .panel-heading
+            .panel-title#contact_success
+          %ul.list-group#contact_errors
+        =form_tag '/contact', method: :post, remote: true, id: 'contact_form' do
+          .row.control-group
+            .form-group.col-xs-12.floating-label-form-group.controls
+              =label_tag 'name', 'Name'
+              =text_field_tag 'name', nil, placeholder: "Name", class: 'form-control',
+                required: "required", "data-validation-required-message" => t('contact_enter_name')
+          .row.control-group
+            .form-group.col-xs-12.floating-label-form-group.controls
+              =label_tag 'email', 'Email Address'
+              =text_field_tag 'email', nil, placeholder: "Email Address", class: 'form-control',
```

Comment:
The = symbol should have one space separating it from code<br>Line is too long. [96/80]